index.js

```diff
@@ -107,6 +109,71 @@ function xp(message) {
     }
 }
 
+client.on("guildMemberAdd", async (member) => {
+    if(!captchaEnable) return;
+    const captcha = new Captcha();
+    captcha.async = true;
+    captcha.addDecoy();
+    captcha.drawTrace();
+    captcha.drawCaptcha();
+
+const verifyEmbed = new Discord.MessageEmbed()
+.setColor('#024F33')
+.setAuthor("Member Joined")
+.setDescription(`Member \`${member.user.tag}\` successfully verified`)
+.setThumbnail(member.user.avatarURL())
+.addField("Member ID", `${member.id}`)
+.addField('Join Date', `${member.joinedAt}`)
+.setTimestamp()
+.setFooter(`${footer} - Made by Donny`);
+
+const verifyEmbed1 = new Discord.MessageEmbed()
+.setColor('ORANGE')
+.setAuthor(`${member.user.tag}`, member.user.avatarURL())
+.setDescription(`Kicked for failing to verify within the ${captchaTIMEms} ms limit`)
+.setTimestamp()
+.setFooter(`User ID: ${member.user.id}`, member.guild.iconURL());
+
+    const captchaAttachment = new MessageAttachment(
+        await captcha.png,
+        "captcha.png"
+    );
+
+    const captchaEmbed = new Discord.MessageEmbed()
+    .setDescription("Please Complete This Captcha")
+    .setImage('attachment://captcha.png')
+    .setFooter(`${footer} - Made By Donny`)
+    .setTimestamp()
+    const verifyDM = member.guild.channels.cache.find(c => c.id === captchaDMCLOSEDID);
+
+    const msg = await member.send({files: [captchaAttachment], embeds: [captchaEmbed],}).catch(() => verifyDM.send(`<@${member.id}> Please enable your dms so I can send you the captcha, one you have enabled your dms rejoin the server. https://media.discordapp.net/attachments/682375042570780830/682407020795920465/rLBBFm8iCy.gif`));
+
+
+    const filter = (message => {
+        if(message.author.id !== member.id) return;
+        if(message.content == captcha.text) return true;
+        else member.send("Wrong Captcha").catch(() => verifyDM.send(`<@${member.id}> Please enable your dms so I can send you the captcha, one you have enabled your dms rejoin the server. https://media.discordapp.net/attachments/682375042570780830/682407020795920465/rLBBFm8iCy.gif`));
```

Comment:
I could be wrong but I believe this else statement won't work since the if statement before it has a semicolon after it